src/main/scala/viper/server/VerificationWorker.scala

```diff
@@ -27,568 +27,564 @@ import scala.language.postfixOps
 
 // Implementation of the Reporter interface used by the backend.
 class ActorReporter(private val actor_ref: ActorRef, val tag: String)
-  extends viper.silver.reporter.Reporter {
+    extends viper.silver.reporter.Reporter {
```

Comment:
Please make sure that your IDE doesn't change the indentation; this complicates the review of the PR. We should use an indentation of 2 spaces fro Scala code.